contracts/VoucherHub.sol

```diff
@@ -164,6 +164,7 @@ contract VoucherHub is
         uint256 voucherType,
         uint256 value
     ) external onlyRole(MINTER_ROLE) returns (address voucherAddress) {
+        require(value > 0, "VoucherHub: no value");
```

Comment:
Maybe say something about the function name => can't create/mint voucher with no value ?

Reply:
```suggestion
        require(value > 0, "VoucherHub: cannot create voucher without value");
```

Reply:
Yes see [Update voucher creation without value revert message & add tmp var](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/32/commits/44490b12fb1b391a03e48fe1b8932e8f6dd6d32b) :)